server/services/sqs-messaging/sqs-send-message.js

```diff
@@ -0,0 +1,18 @@
+const AWS = require('aws-sdk')
+
+const getInitialisationAttributes = config => ({
```

Comment:
We tend to use hoistable functions to help readability, i.e.

`function getInitialisationAttributes (config) {
  ...
}
`

As this extends settings in the config I might be tempted to push the configuration down to the config object where the region can be set and validated by the Joi schema

 i.e. add a `paymentQueueSender` config that can used directly in the sendMessage function.